app/services/follow_remote_account_service.rb

```diff
@@ -75,6 +75,7 @@ def account_needs_webfinger_update?(account)
 
   def get_feed(url)
     response = http_client(write: 20, connect: 20, read: 50).get(Addressable::URI.parse(url).normalize)
+    raise "Feed attempt failed for #{url}: HTTP #{response.code}" unless response.code == 200
```

Comment:
I think this should be raised as a Goldfinger::Error because we already rescue_from those in ApiController